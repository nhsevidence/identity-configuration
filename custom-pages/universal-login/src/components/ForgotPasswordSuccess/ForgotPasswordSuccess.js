import React from 'react'
import { Link } from 'react-router-dom'
import { hideNav } from '../../helpers'
import Nav from "../Nav/Nav";

const ForgotPasswordSuccess = () => {
  hideNav()
  return (
    <div>
      <Nav/>
      <h3>Success</h3>
      <h5 data-qa-sel="forgotPasswordSuccess-message1">
        If that email address is in our records, we will send an email to you
        containing instructions on how to reset your password. You may have to
        wait a few minutes for the email. If you canâ€™t find the email, please
        check your spam and deleted items folders.
      </h5>
      <h5 data-qa-sel="forgotPasswordSuccess-message2">Please act on the email straight away.</h5>
      <Link data-qa-sel="forgotPasswordSuccess-link-to-login" to="/"> Return to Sign in </Link>
    </div>
  )
}

export default ForgotPasswordSuccess