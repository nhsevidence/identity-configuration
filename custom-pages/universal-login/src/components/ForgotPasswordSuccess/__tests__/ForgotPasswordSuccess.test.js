import React from 'react'
import Enzyme, { shallow } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
import ForgotPasswordSuccess from '../ForgotPasswordSuccess'

Enzyme.configure({ adapter: new Adapter() });

describe('ForgotPasswordSuccess components', () => {
  it('should render ForgotPasswordSuccess correctly', () => {
    expect(shallow(<ForgotPasswordSuccess />)).toMatchSnapshot()
  })
});
