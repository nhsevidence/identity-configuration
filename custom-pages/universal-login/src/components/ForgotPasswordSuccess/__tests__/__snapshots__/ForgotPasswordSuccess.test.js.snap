// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ForgotPasswordSuccess components should render ForgotPasswordSuccess correctly 1`] = `ShallowWrapper {}`;
